### PR TITLE
Mentors part 2

### LINKED ISSUE #112: Mentors Section Part 2
- [ ] Style the images part of the mentors stuff (basically the rest of the section) 
- [ ] Note on mobile there are the cactus' - these represent the different "pages" of mentors since we can't fit them all in 3 rows on mobile

Screenshot 2023-01-06 at 12 20 08 PM
Screenshot 2023-01-06 at 12 20 23 PM

### PR DESCRIPTION
**Pull request by deanframe**
Finishes styling and pagination for mentors section of live site

Fixes #112 

Changelist:

- adds styles for the rest of the mentors section according to wireframe
- pagination for mobile

Notes:

- should work at all resolutions
- added dummy data to airtable

Tested:

- locally FF + chrome

### CODE REVIEW

**Review by surbhi-gulati — CHANGES_REQUESTED** [review]
LGTM with spacing nitpicks:

Can you add a little bit more horizontal spacing specifically just in middle sizes: 
image

Slightly extra space on left in the mobile version:
image


And finally, slightly more space before images in the smaller sizes so it doesn't get too close to dropdowns
image

File: live-site/components/mentors-section/MentorsSection.styles.tsx [hunk 1]
```diff
@@ -62,10 +69,96 @@ const StyledMentorsFilterRadio = styled.div`
...
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding-bottom: 20px;
+  justify-content: center;
+  max-width: 95%;
+  margin: auto;
+
+  @media ${max.tabletLg} {
+    max-width: 500px;
```
Comment (jzhzhang66) [inline live-site/components/mentors-section/MentorsSection.styles.tsx]: nitpick but could we change everything to use em's to be standard with the rest of our repo?

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
dean this is AWESOMEEEEe thank you so much for working on this!!

+1 to all of Surbhi's comments and left a minor nitpick but otherwise looks GREAT!!!

File: live-site/components/mentors-section/MentorsSection.styles.tsx [hunk 2]
```diff
@@ -62,10 +69,100 @@ const StyledMentorsFilterRadio = styled.div`
...
+  }
+`
+
+const StyledMentorName = styled.div`
+  color: ${colors.WHITE};
+  font-family: ${fonts.nunitoSansSemibold};
+  font-size: 16px;
+
+  @media ${min.desktop} {
+    font-size: 22px;
```
Comment (jzhzhang66) [inline live-site/components/mentors-section/MentorsSection.styles.tsx]: SORRY extra nit pick but could we please change all the px's to em's? Including font sizes 

https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]

**Review by jzhzhang66 — APPROVED** [review]
APPROVING w comments to change px to em's for font sizes

File: live-site/components/mentors-section/MentorsSection.styles.tsx (hunk 2 above)
Comment (deanframe) [reply live-site/components/mentors-section/MentorsSection.styles.tsx]: OOF i didnt see those

**Comment by surbhi-gulati** [comment]
LGTM!!

**Review by surbhi-gulati — APPROVED** [review]

**Pull request merged**